{
  "comments": [
    {
      "key": {
        "uuid": "f8ce1791_94d8f02d",
        "filename": "src/config/api-server/vnc_cfg_api_server/tests/resources/test_host_based_service.py",
        "patchSetId": 8
      },
      "lineNbr": 68,
      "author": {
        "id": 10
      },
      "writtenOn": "2019-10-04T08:44:50Z",
      "side": 0,
      "message": "We should be able to continue to use other network type, other1, other2, ..., other100",
      "range": {
        "startLine": 68,
        "startChar": 56,
        "endLine": 68,
        "endChar": 61
      },
      "revId": "8370cd0d83daabd5e2d773a858e7cf78f9e646d9",
      "serverId": "c72cd27d-1743-4a55-84a4-cd13b8400c52",
      "unresolved": true
    }
  ]
}